### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by Marcpezin**
I have recreated the readme file. It has been created at the very beginning of XCP-ng and was incomplete. 

Looking for comments to improve.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,67 @@
...
+
+#### Platform
+
+A combination of multiple components to deliver a cohesive solution
+
+**Compute** + **Network** + **Storage** + **API** (Xen API) = **XCP-ng**
+
+#### Open Source
+
+XCP-ng is open source (mostly GPLv2), but also that all the components, documentation, repository and dev process are publicly available and easily accessible.
```
Comment (stormi) [inline README.md]: "that" seems out of place in the sentence

File: README.md [hunk 2]
```diff
@@ -1,9 +1,67 @@
...
+* [Documentation](#documentation)
+* [Contribution & community](#contribution-and-community)
+* [Pro support](#pro-support)
 
-1. The main goal is to be able to enjoy Xenserver power (XAPI/features) with a real community backed solution (not "one company dependent")
-2. We want this to be 100% compatible with XenServer (as possible): ie being able to transfer VMs from XS to XCP-ng and vice-versa
-3. Same for XAPI to be 100% compatible with Open Source management solutions (Xen Orchestra)
+![](misc/assets/logo-glossy-small.png)
+
+## What is XCP-ng ?
```
Comment (stormi) [inline README.md]: No space before `?` (and check the anchor in line 4 because it will probably change)

File: README.md [hunk 3]
```diff
@@ -1,9 +1,67 @@
...
+
+XCP-ng is a virtualization platform based on **Xen Source** and **Citrix® Hypervisor** (formerly XenServer).
+XCP-ng stands for **Xen Cloud Platform - New Generation** and is an *hommage* to the old Open Source project XCP that was ditched when XenServer became Open Sourced in 2013.
```
Comment (stormi) [inline README.md]: * "hommage" => "homage" or "tribute"?
* I'd avoid the word "ditched" which is slang. 
* "became Open Sourced" => "was open-sourced"?

File: README.md [hunk 4]
```diff
@@ -1,9 +1,67 @@
...
+
+### Genesis
+
+* 2003 : Xen project birth, first open source hypervisor
+* 2007 : Xen Source Inc. acquired by Citrix, birth of XenServer (closed source) 
+* 2010 : Citrix strategy focused on virtual desktop (XenApp/XenDesktop) instead of servers virtualization
+* 2011 : XCP 1.0 (Xen Cloud Platform, Open Source XenServer)
+* 2013 : XenServer become Open (end of XCP). Birth of Xen Orchestra
+* 2017 (dec.) : Citrix changes it's pricing strategy regarding XenServer and remove essential features from free XenServer version. New features developed are (mostly) not open source anymore
+* 2018 (janv) : XenServer fork,  birth of XCP-ng
```
Comment (stormi) [inline README.md]: double space before "birth"

**Comment by Marcpezin** [comment]
I fixed your comments, if you are ok with the content, we can ask @Fohdeesha to make a grammar review of this :)

**Comment by Fohdeesha** [comment]
Grammar /spelling edits made directly to source branch

**Comment by Marcpezin** [comment]
Ready to merge when you want @stormi  :)

**Pull request merged**